qbittorrent_lsio.json

```diff
@@ -0,0 +1,58 @@
+{
+	"qBittorrent": {
+		"website": "http://www.qBittorrent.org/",
+		"version": "1.1",
+		"description": "qBittorrent client with a WebUI.<p>Note: Ensure that the WebUI Port is the same as the WEBUI_PORT to avoid issues with the Cross-Site Request Forgery (CSRF) protection that qBittorrent has implemented.<p>The initial WebUI user is <code>admin</code> and initial password is <code>adminadmin</code>. This can then be adjusted in the Settings Menu later.</p><p>For a more detailed description, check here <a href='WebUI Port and CSRF' target='_blank'>https://docs.linuxserver.io/images/docker-qbittorrent#webui_port-variable</a>.</p><p>Based on a custom docker image: <a href='https://hub.docker.com/r/linuxserver/qbittorrent' target='_blank'>https://hub.docker.com/r/linuxserver/qbittorrent</a>, available for amd64 and arm64 architecture.",

```

Comment:
@Hooverdan96 with further testing using 8080 we do get the Web-UI. Likely we have something amiss here I think. Or we need to specify that the Rock-on will only work with 8080!
![8080-Web-ui-ok](https://user-images.githubusercontent.com/2521585/173231455-691520a3-5e51-4b34-956c-285d647cfb62.png)
![8080-web-ui-8080](https://user-images.githubusercontent.com/2521585/173231513-3e9499db-92e7-49cd-a4dd-b202cf4813ec.png)

Reply:
@phillxnet , the image docs actually do support your finding:

> WEBUI_PORT variable
> Due to issues with CSRF and port mapping, should you require to alter the port for the webui you need to change both sides of the -p 8080 switch AND set the WEBUI_PORT variable to the new port.
For example, to set the port to 8090 you need to set -p 8090:8090 and -e WEBUI_PORT=8090
"linuxserver/qbittorrent - LinuxServer.io" https://docs.linuxserver.io/images/docker-qbittorrent

That makes it tricky for us to account for that, though. We could set the port mapping to an exotic port unlikely to be used by another Rock-on but we still need a Port option in the Rock-on wizard so that Rockstor can build the Webui link.

Reply:
@FroggyFlox Agreed. So we likely also want to have in the port text that it must be a set port! That's a bit limiting. But at least we can get it in and working that way until we can handle this type of thing - down the road.

### Port suggestions on this one?

How about 5345 (centermeter range (ish) of cubit ;) currently from:
https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xml
we have:

> 5345-5348		Unassigned

Reply:
I'm OK with 5345.

I believe we already have encountered this situation before and this was how we dealt with it.

Reply:
@phillxnet, @FroggyFlox - thanks for noticing that. I had read and interpreted that by putting the external port and the WEBUI_PORT to the same one, it would be sufficient, but ... of course that was incorrect, because as you pointed out, the port has to be changed in three places, one of which can't be exposed as it's an attribute in the Rockon definition.
I'm ok with that port 5345 as well, and in turn, we don't have to expose the WEBUI_PORT environment variable either.
I will make the changes shortly and update the PR. As always, thanks for the eagle-eyed attention to details that you two provide!